eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/DateAndTimeConverter.php

```diff
@@ -112,6 +112,7 @@ public function toFieldDefinition(StorageFieldDefinition $storageDef, FieldDefin
                 $data = array(
                     'rfc850' => null,
                     'timestamp' => time(),
```

Comment:
@andrerom For our code it doesn't matter, timestamp isn't accessed when timestring is set. The idea was to keep timestamp as is for BC.